resources/units/aircraft/SK-60.yaml

```diff
@@ -0,0 +1,23 @@
+carrier_capable: false
+description:
+  The Saab 105 is a Swedish high-wing, twinjet trainer aircraft developed in the early 1960s 
+  as a private venture by Saab AB.[2] The Swedish Air Force, which had opted to procure the type 
+  for various roles, issued the aircraft with the designation Sk 60. The Sk 60 entered service in 1967, 
+  replacing the ageing De Havilland Vampire fleet.
+  SK-60B is a two-seat attack version for the Swedish Air Force, modified from Sk 60A with new weapons sight
+  
+introduced: 1968
```

Comment:
Interesting, the description states it entered service in 1967, but you're actually taking 1968 😅 
Any particular reason for this?

Reply:
Yea, SK-60 entered service in 67 and the SK-60B light attack came out the next year, which is the one we are using.